pyproject.toml

```diff
@@ -8,5 +8,3 @@ max-line-length = 127
 max-line-length = 127
 per-file-ignores = ['__init__.py:F401']
 
-[tool.cibuildwheel]

```

Comment:
Why was this removed? This will make it build for CPython 6,7,8,9, which is not possible as far as I am aware.

Reply:
Sorry it should not be removed. The merge reverted a few things backward and I missed this change. I'll fix it.